src/routes/v2/organisations.js

```diff
@@ -37,12 +41,13 @@ module.exports = [
             },
             tags: ['api', 'v2'],
             response: {
-                //   schema: OrgModel.externalDefintion,
+                schema: OrgModel.externalDefintion,
+                modify: true
             },
         },
-        handler: (request, h) => {
+        handler: async (request, h) => {
             const { db } = h.context
-            const organisation = Organisation.findByUuid(orgUuid, true, db)
+            const organisation = await Organisation.findByUuid(orgUuid, true, db)
```

Comment:
If we're doing this service layer I'd rather see a generic find such as
`await Organisation.find({ uuid: orgUUid }, true, db)`
or similiar as you've done with the name in the other code, a filter object { filter: { uuid: orgUuid } }

Reply:
Right. The reason this was "seperated" from the other `.find()` was the extra code to include users, and as I see we only need that for single organisations.

Reply:
A suggestion is that we either fetch them in a micro-service similiar way such as
`const organisation = await Organisation.find(.....)
const users = await User.find({ filter: { organisation: org.uuid } })
return {
     organisation,
     users
}
`

or, we add some "include" possibility as I've seen in some cases other projects use (will add complexity)

example:

`
const organisation = await Organisation.find( { filter: { .... }, include: ["user"] })
`

Personally I'd prefer the first way as it's more straight forward even if the second one would basically do the same internally but would increase our code complexity and increase coupling. What do you think @Birkbjo ?

Reply:
Hmm... I would expect at least a reference in the organisation or the user objects. Maybe the Organisation's `.find()` method could call the user.find and append there? Then we would need the include option. Or do we want services to be independent of each other?

It depends on what kind of API we want, but getting a normalized structure like that is not what I would expect from a resource.

Reply:
Another reason is that findByUuid always returns one single object (or throws if not exists). And thus nicely maps to the respective endpoints, without the need to check for an empty collection in the case of the `uuid endpoint`. 

You could argue that we shouldn't couple the services to the handlers, and I agree that we should try not to do that, but it's a nice bonus if it's natural mapping. I feel like the distinction between "get a single object by the identifier" and "get a list with this criteria" is distinct enough to have them as different functions.
However I don't really like the "includeUsers" parameter-solution either. Maybe we should move it to an options object or something.

Reply:
A suggestion about findByUuid is to rename it to "findOne", "one", "first" or "single" to make it even clearer that it only returns one result, and then we can still use the same filter/config object to pass either uuid, name, or whatnot we want to use in the where-part of the query.